src/cookies.ts

```diff
@@ -61,8 +65,8 @@ export const setCookie = ({
 }): void => {
 	const expires = new Date();
 
-	if (!isValidCookieValue(name) || !isValidCookieValue(value)) {
-		return;
+	if (!isValidCookie(name, value)) {
+		throw new Error(`${ERR_INVALID_COOKIE} .${name}=${value}`);
```

Comment:
what is the `.` doing here?

Reply:
Not much I was following the format from: [cookie.ts](https://github.com/guardian/dotcom-rendering/blob/d00f45c9b6be29519b056885b0be4d818fc71b67/src/web/browser/cookie.ts#L80)

I can remove it.